serverlessworkflow/modules/ROOT/pages/service-orchestration/orchestration-of-grpc-services.adoc

```diff
@@ -1 +1,164 @@
+
+----
+// The greeter service definition.
+service Greeter {
+  // Sends a greeting
+  rpc SayHello (HelloRequest) returns (HelloReply) {}
+}
+
+// The request message containing the user's name.
+message HelloRequest {
+  string name = 1;
+  string language=2;
+}
+
+// The response message containing the greetings
+message HelloReply {
+  string message = 1;
+}
+----
+
+
+=== Workflow definition file
+
+The flow defines a RPC link:https://github.com/serverlessworkflow/specification/blob/main/specification.md#using-functions-for-rpc-service-invocations[function]. 
+[source,json]
+----
+
+"functions": [
+    {
+      "name": "sayHello",
+      "type": "rpc",
+      "operation": "greeting.proto#Greeter#SayHello"
+    }
+  ]
+----
+
+The `operation` property is composed of three tokens separated by #: the URI of the proto file, the name of the service, and the name of the method. Kogito supports three URI schemes: `http`, `file`, and `classpath`, the latter being the default. 
+Therefore, in this case, the flow declares that `greeting.proto` is expected to be found in the project classpath, that the service to be used is `Greeter` and the method name is `sayHello` 
+
+The flow execution consists of a single link:https://github.com/serverlessworkflow/specification/blob/main/specification.md#operation-state[operation] state, composed of one https://github.com/serverlessworkflow/specification/blob/main/specification.md#action-definition[action].
+
+[source,json]
+----
+ {
+    "name": "sayHello",
+    "functionRef" : {
+        "refName": "sayHello",
+        "arguments": {
+            "name": ".name",
+            "language": ".language"
+        }
+    }
+}
+----
+
+The action invokes the `sayHello` RPC method using as parameters two model variables: `name` and `language`, provided as part of the REST call that starts the flow. 
+ 
+=== Application properties
+
+We need to tell the application where the gRPC server is located.
+
+[source,properties]
+----
+quarkus.grpc.clients.Greeter.host=localhost
+quarkus.grpc.clients.Greeter.port=50051
+----
+
+Internally, the Kogito Serverless Workflow implementation uses the same set of properties as link:https://quarkus.io/guides/grpc-service-consumption#client-configuration[Quarkus gRPC client].
+The "client-name" used by Kogito is the service name declared in the proto file. `Greeter` in this case.
+
+Why is port 50051? In the same repo as the workflow application, a link:https://github.com/kiegroup/kogito-examples/tree/main/kogito-quarkus-examples/serverless-workflow-greeting-rpc-quarkus/serverless-workflow-greeting-server-rpc-quarkus[maven project] is available, which provides a simple implementation of the greeter service. By default, this gRPC server runs on port 50051, hence the value for the port property. 
```

Comment:
isn't repo word confusing here?
maybe same project or the default port use by quarkus grpc extension?

Reply:
I mean github repository, will change that